framework/Samplers/Sampler.py

```diff
@@ -580,6 +580,9 @@ def initialize(self,externalSeeding=None,solutionExport=None):
     # Register expected metadata
     meta = ['ProbabilityWeight','prefix','PointProbability']
     for var in self.toBeSampled.keys():
+      #if self.variables2distributionsMapping[var]['totDim'] > 1:
+      #  meta +=  ['ProbabilityWeight-'+ var.replace(",","-")]
+      #else:
```

Comment:
dead code

Reply:
I left it there bacause what we were discussing today about the ND sampling ...that I honestly do not understand how we handle in the statistics (there is this replace for correlated dimensions but I do not see any special case in the basic statistics). @mandd and @wangcj05  any light here?